### PR TITLE
Igluctl 0.12.2

### PR DESCRIPTION
**Pull request by spenes**

### CODE REVIEW

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ Iglu provides a CLI application, called igluctl which allows you to perform most
...
     - `push` - push set of JSON Schemas from static registry to full-featured (Scala Registry for example) one
     - `pull` - pull set of JSON Schemas from registry to local folder
-    - `parquet-verify` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks).  If the schema is not evolved correctly, the offending version will be stored in a separate column with `_recovered_` in its name.
+    - `verify-parquet` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
+    - `verify-redshift` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for loading into Redshift. It reports the schema criterions which belongs to breaking schema versions.
```
Comment (stanch) [inline docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:23]: > schema criterions which belongs to breaking schema versions

Not sure I understand this part. Is there a chance we could avoid using the words “schema criterions”? I suspect very few people know what they mean

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md [hunk 2]
```diff
@@ -268,6 +269,31 @@ Example:
...
+
+`igluctl static verify-parquet` verifies that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
+
+It accepts one required arguments:
+
+- `input` - path to your schema files.
+
+```bash
+$ ./igluctl static verify-parquet /path/to/static/registry
+```
```
Comment (stanch) [inline docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:282]: Could you add example output?

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md [hunk 3]
```diff
@@ -268,6 +269,31 @@ Example:
...
+
+## static verify-redshift
+
+`igluctl static verify-redshift` verifies that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for loading into Redshift. It reports the schema criterions which belongs to breaking schema versions.
+
+It accepts two required arguments:
+
+- `server` - Iglu Server URL.
+- `apikey` - Iglu Server Read ApiKey
```
Comment (stanch) [inline docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:291]: Why does it take these args if it’s `static`?

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md [hunk 4]
```diff
@@ -268,6 +269,31 @@ Example:
...
+
+```bash
```
Comment (stanch) [inline docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:293]: Could you add example output?

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 1 above)
Comment (spenes) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:23]: `verify-redshift`'s output is like this:
```
iglu:com.snowplowanalytics.iglu/resolver-config/jsonschema/1-*-*
iglu:com.snowplowanalytics.snowplow/event_fingerprint_config/jsonschema/1-*-*
iglu:com.snowplowanalytics.snowplow.badrows/loader_runtime_error/jsonschema/1-*-*
```
We might say `It reports the breaking schema versions` instead even though it is not exactly true.

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 3 above)
Comment (spenes) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:291]: Seems like it was mistake. If you want, I can open a ticket to fix this. If we had to fix it, should we wait to update the docs until the fix ?

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 1 above)
Comment (stanch) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:23]: Maybe “it reports the major schema versions within which schema evolution rules were broken”?

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 3 above)
Comment (stanch) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:291]: Maybe we can just leave a note in the docs that “static” is a misnomer and will be fixed in the next version?

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 2 above)
Comment (stanch) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:282]: Just checking — you would still add output right? (Not just the command example.)

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 4 above)
Comment (stanch) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:293]: Just checking — you would still add output right? (Not just the command example.)

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 4 above)
Comment (spenes) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:309]: Yep, added it in the new commit.

File: docs/pipeline-components-and-applications/iglu/igluctl-2/index.md (hunk 2 above)
Comment (spenes) [reply docs/pipeline-components-and-applications/iglu/igluctl-2/index.md:284]: Yep, added it in the new commit.

**Review by stanch — APPROVED** [review]

**Pull request merged**